lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -28,6 +28,30 @@ class SpecificMatcher < Base
           MSG = 'Prefer `%<good_matcher>s` over `%<bad_matcher>s`.'
           RESTRICT_ON_SEND = %i[have_selector have_no_selector have_css
                                 have_no_css].freeze
+          HAVE_BUTTON_OPTIONS = %w[above below left_of right_of near count
+                                   minimum maximum between text id class
+                                   style visible obscured exact exact_text
+                                   normalize_ws match wait filter_set focused
+                                   disabled name value title type name
+                                   disabled].freeze
+          HAVE_LINK_OPTIONS = %w[above below left_of right_of near count
+                                 minimum maximum between text id class style
+                                 visible obscured exact exact_text normalize_ws
+                                 match wait filter_set focused href href
```

Comment:
Double `href`.
What makes me wonder if there are other duplicates.

Reply:
Oops, I reviewed the other definitions as well.

Reply:
I was referring to the capybara error message. but it appears to have been a duplicate suggestion display.
https://github.com/teamcapybara/capybara/pull/2572